### DISCUSSION TITLE
ZeroSumNormal sigma dim >1

### DISCUSSION THREAD

**Discussion by bernardocaldas** [pymc-devs/pymc3#7272] [Ideas]
Any plans to enable ZeroSumNormal to have different sigmas across zerosum dimensions?

**Comment by ricardoV94** [comment]
This was discussed internally many times, but I don't know the details 

CC @aseyboldt @lucianopaz @AlexAndorra

**Comment by AlexAndorra** [comment]
Yep, definitely needed. We could work on that while we work on the PR for nested hierarchies @lucianopaz

**Reply by ricardoV94** [reply]
Wasn't there some ambiguity as to what variable sigma across zerosum axes means, or am I misremembering it?

**Comment by AlexAndorra** [comment]
Yeah it’s not trivial IIRC

Il giorno dom 21 apr 2024 alle 18:15 Ricardo Vieira <
***@***.***> ha scritto:

**Comment by bernardocaldas** [comment]
What could a newbie like me help with?
